main/helpers/fileParser.ts

```diff
@@ -0,0 +1,93 @@
+import fs from "fs";
+import { getPathFromDate, calcDurationBetweenTimes } from "../helpers/datetime";
+export type Activity = {
+  acti: string;
+  desc: string;
+  dur: number;
+};
+
+export type AllActivities = {
+  internal: Activity[];
+  hr: Activity[];
+};
+
+export function fileParser(reportsFolder: string, date: Date): AllActivities {
```

Comment:
this is a common name, can you make mo detailed name like "parseReportsInfo" or "parseActivityDescription". Also you function have to start from verb.